askama_shared/src/i18n_runtime.rs

```diff
@@ -0,0 +1,322 @@
+//! Code used in the implementation of `impl_localize!` and the `localize()` filter.
+//!
+//! Everything in this module should be considered an internal implementation detail; it is only public
+//! for use by the macro.
+//!
+//! Maintenance note: in general, the policy is to move as much i18n code as possible into here;
+//! whatever absolutely *must* be included in the generated code is done in askama_derive.
+
+use accept_language::parse as accept_language_parse;
+use fluent_bundle::{FluentBundle, FluentResource, FluentValue};
+use std::collections::HashMap;
+
+use super::{Error, Result};
+
+pub use lazy_static::lazy_static;
+
+/// An I18n argument value. Instantiated only by the `{ localize() }` filter.
```

Comment:
I prefer to have code ordered the other way around: high-level complex dependent API up top, simple private types that don't depend on anything somewhere down the bottom. It's explained a bit more here:

https://deterministic.space/how-to-order-rust-code.html

Reply:
Huh, I never thought about this. Can do.